{"id":26053,"className":"java.security.cert.CertPathValidatorException","methodName":"getReason","signature":"\u003cjava.security.cert.CertPathValidatorException: java.security.cert.CertPathValidatorException$Reason getReason()\u003e","javadoc":"/** \n * Returns the reason that the validation failed. The reason is associated with the index of the certificate returned by {@link #getIndex}.\n * @return the reason that the validation failed, or\u003ccode\u003eBasicReason.UNSPECIFIED\u003c/code\u003e if a reason has not been specified\n * @since 1.7\n */\n","tf":{"freq":{"valid":2,"certif":1,"index":2,"fail":2,"basic":1,"associ":1,"reason":6,"link":1,"specifi":1},"maxFreq":6,"totalTerms":17},"loc":3}